Christian particle and white box

ChristianTW opened the pull request:
I made particle effects for Scavenge melee, dot, and life steal, and for Splendor melee, aoe, and crit/high damage hit. I also modified whitebox area 5 and added 2 new whitebox areas.

Commit:
Added some test particles and materials

Commit:
Added some more scavenge effects

Commit:
Added particles for scavenge melee, dot, and life steal, and splendor melee, aoe, and crit

Commit:
Added 2 new test whiteboxes

Aiden-Thacker commented:
There are a few changes we would like to see before pushing to main. First could we get a connect of two different areas in your whit box scene could we get Area 04(1) connected to the Area with the Monster Skull. Now onto your particles great job we only have a few changes we want you to make. First Liam think Scavenge could be a little more blue. Second we believe ScavengeDOT(2) should become the SplendorAOE instead (looks great). Finally programming Xander is asking if you could make SmokeBurst(4-6) one of these four to be the new ScavengeDOT, and what I mean by this is at the end of the particle that currently plays make it into a circle shape. For the last part please DM Xander, he can explain better than I can.

Commit:
Set up a doorway between two test areas, turned on experimental probuilder

I have a tunnel and doors for the test areas, ass well as cut outs in the walls so the player can pass through the tunnel when the door is moved

Commit:
Added falling particles to scavenge dot and modified scavenge dot2 into a splendor aoe
